### PR TITLE
Build Metal3 components during IPA building

### PR DESCRIPTION
**Pull request by Rozzii**

### CODE REVIEW

**Review by kashifest — APPROVED** [review]

File: ci/scripts/image_scripts/start_centos_fullstack_build.sh [hunk 1]
```diff
@@ -86,8 +112,13 @@ ssh \
...
-  BMO_BRANCH="${BMO_BRANCH:-main}" BMO_COMMIT="${BMO_COMMIT:-HEAD}" \
+  IPA_REPO="${IPA_REPO}" IPA_BRANCH="${IPA_BRANCH}" IPA_REF="${IPA_REF}" \
+  IPA_BUILDER_REPO="${IPA_BUILDER_REPO}" IPA_BUILDER_BRANCH="${IPA_BUILDER_BRANCH}" IPA_BUILDER_COMMIT="${IPA_BUILDER_COMMIT}" \
+  METAL3_DEV_ENV_REPO="${METAL3_DEV_ENV_REPO}" METAL3_DEV_ENV_BRANCH="${METAL3_DEV_ENV_BRANCH}" METAL3_DEV_ENV_COMMIT="${METAL3_DEV_ENV_COMMIT}" \
+  BMO_REPO="${BMO_REPO}" BMO_BRANCH="${BMO_BRANCH}" BMO_COMMIT="${BMO_COMMIT}" \
+  CAPM3_REPO="${CAPM3_REPO}" CAPM3_BRANCH="${CAPM3_BRANCH}" CAPM3_COMMIT="${CAPM3_COMMIT}" \
+  IPAM_REPO="${IPAM_REPO}" IPAM_BRANCH="${IPAM_BRANCH}" IPAM_COMMIT="${IPAM_COMMIT}" \
+  CAPI_REPO="${CAPI_REPO}" CAPI_BRANCH="${CAPI_BRANCH}" CAPI_COMMIT="${CAPI_COMMIT}" \
+  BUILD_CAPM3_LOCALLY="${BUILD_CAPM3_LOCALLY}" BUILD_BMO_LOCALLY="${BUILD_BMO_LOCALLY}" \
+  BUILD_IPAM_LOCALLY="${BUILD_IPAM_LOCALLY}" BUILD_CAPI_LOCALLY="${BUILD_CAPI_LOCALLY}" \
```
Comment (furkatgofurov7) [inline ci/scripts/image_scripts/start_centos_fullstack_build.sh:123]: `BUILD_BMO_LOCALLY` is missing?

Comment (Rozzii) [reply ci/scripts/image_scripts/start_centos_fullstack_build.sh:123]: It is there L122 at the end

Comment (furkatgofurov7) [reply ci/scripts/image_scripts/start_centos_fullstack_build.sh:123]: I don't see it in this file

**Pull request merged**